### ISSUE TITLE
Template Needed

### ISSUE THREAD

**Issue by bryngo** [Housemate1/Housemate1.github.io#1]
In order to move as fast as possible, I've decided that we should use a template to design our landing page for our website. Many, many of these can be found online.

Skills needed/used: Googling, Frontend

Things to look for when completing this task: We need a template that provides a log in page, log out, register users, etc. For inspiration, checkout Facebook, Instagram, or Twitter's login page. We're going for something like that. 

In summary, the assignee of this task shall decide if we want to use a template, or create a page from scratch. Please have a link to the template posted here or start working on the page (if doing from scratch) by tonight.

**Comment by bryngo** [comment]
@GlendaleAcosta has created a mock up for the login page. We will template the landing page off of this.
